src/Toml/Bi/Combinators.hs

```diff
@@ -254,5 +252,5 @@ table codec key = Codec input output
     handleTableName e                         = throwError e
 
 -- | Used for @newtype@ wrappers.
-wrapper :: forall b a . Coercible a b => (Key -> TomlCodec a) -> Key -> TomlCodec b
-wrapper bi key = dimap coerce coerce (bi key)
+diwrap :: forall b a r w . (Coercible a b, Functor r, Functor w) => BiCodec r w a -> BiCodec r w b
```

Comment:
You can move this function to `Toml.Bi.Monad` module because it's general enough